### PR TITLE
Refactor: move videopress placeholder component to jetpack components

### PR DESCRIPTION
**Pull request by kozer**
Fixes #

## Proposed changes:

*

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to '..'
*

### CODE REVIEW

File: projects/js-packages/components/components/loading-placeholder/stories/index.tsx [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
  */
-import Placeholder from '..';
+import LoadingPlaceholder from '..';
 /**
  * Types
  */
 import type { ComponentStory, ComponentMeta } from '@storybook/react';
 
 export default {
 	title: 'Packages/VideoPress/Placeholder',
```
Comment (jeherve) [inline projects/js-packages/components/components/loading-placeholder/stories/index.tsx]: You may have to update this now.

**Review by jeherve — COMMENTED** [review]
I'll let others more familiar with the components chime in on this, but it may be easier to split this into 2 PRs:

- One that adds the new component, without using it anywhere yet, but that you could check works just fine thanks to the Storybook (run `cd projects/js-packages/storybook && pnpm run storybook:dev` on your local dev env)
- One that starts using it in VideoPress.

**Comment by dhasilva** [comment]
I agree with @jeherve here, better to just add the component to the Components js-package first and test it by running storybook locally.

**Comment by kozer** [comment]
Closed in favor of https://github.com/Automattic/jetpack/pull/29271 and https://github.com/Automattic/jetpack/pull/29270

**Pull request closed without merging**